groups/chd-infrastructure/variables.tf

```diff
@@ -0,0 +1,141 @@
+# ------------------------------------------------------------------------------
+# AWS Variables
+# ------------------------------------------------------------------------------
+variable "aws_region" {
+  type        = string
+  description = "The AWS region in which resources will be administered"
+}
+
+variable "aws_profile" {
+  type        = string
+  description = "The AWS profile to use"
+}
+
+variable "aws_account" {
+  type        = string
+  description = "The name of the AWS Account in which resources will be administered"
+}
+
+# ------------------------------------------------------------------------------
+# AWS Variables - Shorthand
+# ------------------------------------------------------------------------------
+
+variable "account" {
+  type        = string
+  description = "Short version of the name of the AWS Account in which resources will be administered"
+}
+
+variable "region" {
+  type        = string
+  description = "Short version of the name of the AWS region in which resources will be administered"
+}
+
+# ------------------------------------------------------------------------------
+# Environment Variables
+# ------------------------------------------------------------------------------
+variable "environment" {
+  type        = string
+  description = "The name of the environment"
+}
+
+# ------------------------------------------------------------------------------
+# Vault Variables
+# ------------------------------------------------------------------------------
+variable "vault_username" {
```

Comment:
These would have to be moved alongside the relevant provider so as to prompt correctly

Reply:
Updated in commit 9f8899a